index.html

```diff
@@ -57,6 +57,260 @@ <h2>I'm a Software Developer</h2>
           </li>
         </ul>
       </div>
+    </main>
+    <section>
+      <div>
+        <div class="story">
+          <div class="story-title">
+            <h4>My Recent Works</h4>
+          </div>
+          <div class="card-top">
+            <img src="./pics/image1.svg" alt="image one" />
+          </div>
+          <div class="post-title">
+            <h4>Multi-Post Stories</h4>
+          </div>
+          <div class="card-supporting-text">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+          </div>
+          <div class="languages">
+            <ul>
```

Comment:
- [ ] Kindly fix the [issue](https://github.com/Cavein254/mobile-first/pull/2/files#r1148425930) mentioned by the previous reviewer so that you can match the given design. The issue is not solved yet.

- Your Design
- ![image](https://user-images.githubusercontent.com/60083437/227743720-4e30b509-ab97-4d8e-86d1-3a6a62e1d64d.png)

Reply:
![Screenshot_2023-03-26_02-06-19](https://user-images.githubusercontent.com/13707896/227746659-c69b8725-8cf6-4b99-8b79-37c0e48bef72.png)